index.js

```diff
@@ -22,9 +22,17 @@ function zipkinPlugin (fastify, opts, next) {
 
   const ctxImpl = new CLSContext('zipkin')
 
+  let zipkinUrl = `${opts.zipkinUrl}/api/v2/spans`
```

Comment:
I would prefer to add a `zipkinPath` option instead so that things can be combined.

Reply:
Sounds good. I'll make that change.